style.css

```diff
@@ -0,0 +1,52 @@
+h1 {
+    text-align: center;
+    color:#0b0b0b;
+    /* Text-Color */
+    font-size: xx-large;
+    border-style: dotted;
+    
+}
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
+    background-color: #9c489b;
+  }
+  
```

Comment:
Inconsistent whitespace between css blocks, keep it consistent! one line between each style rule.